config.json

```diff
@@ -193,12 +197,12 @@
       "slug": "atbash-cipher",
       "uuid": "82169e85-3938-48da-ada9-7394adadc57a",
       "core": false,
-      "unlocked_by": "rna-transcription",
+      "unlocked_by": "hamming",
       "difficulty": 4,
       "topics": [
         "cipher",
         "string"
       ]
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
Are you still using configlet to format the config?

Reply:
Oh, does `make -j` run `configlet` ?

Reply:
I just branched from upstream/feature after getting yours in. Mm, answer is yes, I'll change the makefile in this pull